contracts/scripts/upgrade_aligned_contracts.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [ -z "$MULTISIG" ]; then
+  echo "Missing MULTISIG env variable"
+  exit 1
+fi
```

Comment:
We also need to check for `OUTPUT_PATH`. We should also react to errors in general. `bash` supports a general way to do so with a flag (I believe it was `set -e`, but I need to check) plus `trap` to handle cleanup.

Reply:
If OUTPUT_PATH is not set, it will fail to execute the forge script.
Anyhow, i have added the explicit check 👍